canonical/ubuntu.com
# Update Canonical short form agreement

## Done
- Version the legal short-terms
- Modify the latest version with the copy changes mentioned in the bug [1]
- Applied the changes to the JA page
- Changed the page structure to match the copy doc
- Updated the copy doc with the product change
## QA
- Check the copy docs [2] structure matches the /legel/short-term section
- Check on the latest version of the terms has "Landscape Dedicated Server" to "Landscape On Premises" and "LDS" to "OPL"
## Issue / Card

[1] https://bugs.launchpad.net/ubuntu-website-content/+bug/1584100
[2] https://drive.google.com/drive/u/1/folders/0B2_PyaH0PaVmVVBJRHZxWFlhbkk
